platform/src/test/java/org/exoplatform/platform/qa/ui/platform/plf/PlfNavigationLeftNavigationTestIT.java

```diff
@@ -0,0 +1,205 @@
+import static org.exoplatform.platform.qa.ui.selenium.locator.gatein.GateinLocator.ELEMENT_LEFT_NAVIGATION_NODE_CHECK;
+import static org.exoplatform.platform.qa.ui.selenium.locator.gatein.GateinLocator.ELEMENT_NAVIGATION_NODE_CHECK;
+import static org.exoplatform.platform.qa.ui.selenium.locator.social.SocialLocator.ELEMENT_SPACE_NAME_LEFT_NAVIGATION;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+import static org.exoplatform.platform.qa.ui.selenium.testbase.LocatorTestBase.ELEMENT_INPUT_USERNAME_CAS;
+import static org.exoplatform.platform.qa.ui.selenium.testbase.LocatorTestBase.ELEMENT_SKIP_BUTTON;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.Configuration;
+
+import org.exoplatform.platform.qa.ui.commons.Base;
+import org.exoplatform.platform.qa.ui.core.context.BugInPLF;
+import org.exoplatform.platform.qa.ui.gatein.pageobject.NavigationManagement;
+import org.exoplatform.platform.qa.ui.gatein.pageobject.PortalManageSites;
+import org.exoplatform.platform.qa.ui.selenium.platform.HomePagePlatform;
+import org.exoplatform.platform.qa.ui.selenium.platform.ManageLogInOut;
+import org.exoplatform.platform.qa.ui.selenium.platform.NavigationToolbar;
+import org.exoplatform.platform.qa.ui.selenium.platform.social.SpaceManagement;
+
+@Tag("sniff")
+@Tag("plf")
+public class PlfNavigationLeftNavigationTestIT extends Base {
+
+  HomePagePlatform     homePagePlatform;
+
+  ManageLogInOut       manageLogInOut;
+
+  SpaceManagement      spaceManagement;
+
+  NavigationToolbar    navigationToolbar;
+
+  PortalManageSites    portalManageSites;
+
+  NavigationManagement navigationManagement;
+
+  @BeforeEach
+  public void setupBeforeMethod() {
+
+    info("Start setUpBeforeMethod");
+    manageLogInOut = new ManageLogInOut(this);
+    homePagePlatform = new HomePagePlatform(this);
+    spaceManagement = new SpaceManagement(this);
+    navigationToolbar = new NavigationToolbar(this);
+    portalManageSites = new PortalManageSites(this);
+    navigationManagement = new NavigationManagement(this);
+
+    if ($(ELEMENT_SKIP_BUTTON).is(Condition.exist)) {
+      $(ELEMENT_SKIP_BUTTON).click();
+    }
+    if ($(ELEMENT_INPUT_USERNAME_CAS).is(Condition.not(Condition.exist))) {
+      manageLogInOut.signOut();
+    }
+    manageLogInOut.signInCas(username, password);
+  }
+
+  @AfterEach
+  public void signout() {
+    manageLogInOut.signOut();
+  }
+  /*
+   * Step Number: 1 Step Name: - Connect to Intranet Step Description: - Login as
+   * a user - Connect to Intranet Input Data: Expected Outcome: - The left
+   * Navigation is displayed - The "COMPANY" list is displayed with applications
+   * in the following order:* Home* Connections* Wiki* Documents* Forums*
+   * Calendar* Other personal pages
+   */
```

Comment:
Comments are not well placed, they should be closed to the below method, not to the above one.